tests/system/small/ml/test_metrics_pairwise.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pandas as pd
+
+from bigframes.ml import metrics
+import bigframes.pandas as bpd
+
+
+def test_cosine_similarity():
```

Comment:
I assume this ML.DISTANCE COSINE in BQML is different from sklearn package `sklearn.metrics.pairwise import cosine_similarity`, are these two comparable?

```
from sklearn.metrics.pairwise import cosine_similarity
X = [[4.1, 0.5, 1.0]]
Y = [[3.0, 0, 2.5]]
cosine_similarity(X, Y)
```

Reply:
Totally different, sklearn takes matrixes and returns matrixes, which we will be hard to support.